src/PIL/ImageFont.py

```diff
@@ -582,22 +582,13 @@ def getmask2(
         _string_length_check(text)
         if start is None:
             start = (0, 0)
-        im = None
-        size = None
 
         def fill(width, height):
-            nonlocal im, size
-
             size = (width, height)
-            if Image.MAX_IMAGE_PIXELS is not None:
-                pixels = max(1, width) * max(1, height)
-                if pixels > 2 * Image.MAX_IMAGE_PIXELS:
-                    return
-
-            im = Image.core.fill("RGBA" if mode == "RGBA" else "L", size)
-            return im
+            Image._decompression_bomb_check(size)
+            return Image.core.fill("RGBA" if mode == "RGBA" else "L", size)
 
-        offset = self.font.render(
+        return self.font.render(
```

Comment:
Before we returned `(im, offset)`, but now just `offset`, is this a breaking change?

Reply:
This changes `self.font.render` to return `(im, offset)` instead of just `offset`.